web-app/js/HighDimensionalData.js

```diff
@@ -350,10 +350,12 @@ HighDimensionalData.prototype.get_inputs = function (divId) {
     ]
 }
 
-HighDimensionalData.prototype.gather_high_dimensional_data = function (divId, hideAggregration) {
+HighDimensionalData.prototype.gather_high_dimensional_data = function (divId, hideAggregration, doValidatePlatforms) {
```

Comment:
The value is true by default if it is not specified. See the third line in this method.